src/Apps/Settings/Routes/MyCollection/Components/MyCollectionEmptyState.tsx

```diff
@@ -7,230 +7,150 @@ import {
   ResponsiveBox,
+  Spacer,
   Text,
 } from "@artsy/palette"
 import { useMyCollectionTracking } from "Apps/MyCollection/Routes/Hooks/useMyCollectionTracking"
-import { EmptyMyCollectionPageProps } from "Apps/Settings/Routes/MyCollection/Components/EmptyMyCollectionPage"
+import { useAuthDialog } from "Components/AuthDialog"
 import { ModalType } from "Components/Authentication/Types"
 import { RouterLink } from "System/Router/RouterLink"
 import { useSystemContext } from "System/SystemContext"
 import { useFeatureFlag } from "System/useFeatureFlag"
-import { openAuthModal } from "Utils/openAuthModal"
 import { resized } from "Utils/resized"
-import { Media } from "Utils/Responsive"
 
-const DesktopLayout: React.FC<EmptyMyCollectionPageProps> = ({
-  loggedOutState,
-}) => {
+export const MyCollectionEmptyState: React.FC = () => {
   const isCollectorProfileEnabled = useFeatureFlag("cx-collector-profile")
 
-  const { mediator } = useSystemContext()
+  const { isLoggedIn } = useSystemContext()
+
   const {
     addCollectedArtwork: trackAddCollectedArtwork,
   } = useMyCollectionTracking()
 
   const image = resized(
     "https://files.artsy.net/images/my-coll-get-app-img.jpg",
-    {
-      width: 770,
-      height: 652,
-    }
+    { width: 910, height: 770 } // TODO: Source image should be 4:3, ideally. Is currently 709:600
   )
 
+  const { showAuthDialog } = useAuthDialog()
+
   return (
-    <GridColumns mb={12} gridRowGap={4} alignItems="center">
-      <Column span={6}>
-        <Text variant="xxl">Manage Your Art Collection Online</Text>
+    <GridColumns gridRowGap={4} alignItems="center">
+      <Column span={6} order={[1, 0]} py={[0, 2]}>
+        <Text variant={["xl", "xl", "xxl"]}>
+          Manage Your Art Collection Online
+        </Text>
 
-        <Text variant="sm" mt={2} mb={6}>
+        <Spacer y={[0, 2]} />
+
+        <Text variant="sm">
           Access price and market insights and build an online record of your
           collection.
         </Text>
-        {!!loggedOutState ? (
-          <Button
-            variant="primaryBlack"
-            onClick={() => {
-              mediator &&
-                openAuthModal(mediator, {
-                  mode: ModalType.login,
-                  intent: Intent.login,
-                  contextModule: ContextModule.myCollectionHome,
-                  copy: "Log in to upload works to My Collection",
-                })
-              trackAddCollectedArtwork()
-            }}
-          >
-            Upload Artwork
-          </Button>
-        ) : (
-          <Button
-            // @ts-ignore
-            as={RouterLink}
-            variant="primaryBlack"
-            to={
-              isCollectorProfileEnabled
-                ? "/collector-profile/my-collection/artworks/new"
-                : "/my-collection/artworks/new"
-            }
-            onClick={() => trackAddCollectedArtwork()}
-          >
-            Upload Artwork
-          </Button>
```

Comment:
No need for the two buttons. Just check in the `onClick` and `preventDefault` to block the link from executing.